### DISCUSSION TITLE
How to make a dataset like YoutubeVIS or video instance segmentation

### DISCUSSION THREAD

**Discussion by Alxx999** [wkentaro/labelme#1058] [Q&A / Help]
Thanks!!!

**Comment by yclicc** [comment]
For guidance for the devs the JSON schema for YoutubeVIS is:

{
            "info" : info,
            "videos" : [video],
            "annotations" : [annotation],
            "categories" : [category],
        }
        video{
            "id" : int,
            "width" : int,
            "height" : int,
            "length" : int,
            "file_names" : [file_name],
        }
        annotation{
            "id" : int, 
            "video_id" : int, 
            "category_id" : int, 
            "segmentations" : [RLE or [polygon] or None], 
            "areas" : [float or None], 
            "bboxes" : [[x,y,width,height] or None], 
            "iscrowd" : 0 or 1,
        }
        category{
            "id" : int, 
            "name" : str, 
            "supercategory" : str,
        }

Where basically several things which are strings in the [COCO spec](https://cocodataset.org/#format-data) become lists of strings corresponding to consecutive frames (actually every 5th frame) of video.

I'm not sure how the code for labelme could be adapted to output (or edit) this format though.
